crates/host/src/wasmbus/mod.rs

```diff
@@ -2000,13 +1993,84 @@ impl Host {
+                                                &health_lattice_prefix,
+                                                "health_check_passed",
+                                                event::provider_health_check(
+                                                    &health_provider_id,
+                                                    &health_link_name,
+                                                    &health_contract_id,
+                                                )
+                                            ).await {
+                                                warn!(?e, "failed to publish provider health check succeeded event");
+                                            }
+                                        },
+                                        (Ok(HealthCheckResponse { healthy: false, ..}), true) => {
+                                            trace!(provider_id=health_provider_id, "provider health check failed");
+                                            previous_healthy = false;
+                                            if let Err(e) = event::publish(
+                                                &event_builder,
+                                                &ctl_nats,
+                                                &health_lattice_prefix,
+                                                "health_check_failed",
+                                                event::provider_health_check(
+                                                    &health_provider_id,
+                                                    &health_link_name,
+                                                    &health_contract_id,
+                                                )
+                                            ).await {
+                                                warn!(?e, "failed to publish provider health check failed event");
+                                            }
+                                        }
+                                        // If the provider health status didn't change, we don't need to publish an event
+                                        (Ok(_), _) => (),
+                                        _ => warn!("failed to deserialize provider health check response"),
+                                    }
+                                }
+                                else {
+                                    warn!("failed to request provider health, retrying in 30 seconds");
+                                }
+                        }
+                        exit_status = child.wait() => match exit_status {
+                            Ok(status) => {
+                                debug!("`{}` exited with `{status:?}`", path.display());
+                                break;
+                            }
+                            Err(e) => {
+                                error!("failed to wait for `{}` to execute: {e}", path.display());
```

Comment:
```suggestion
                                warn!("failed to wait for `{}` to execute: {e}", path.display());
```

No need to spam production logs with errors caused by broken providers (user error)